catalog/ui/catalog-ui-search/src/main/webapp/styles/components/jquery-ui.less

```diff
@@ -1,6 +1,7 @@
 .ui-multiselect-menu {
     .changeBackground(@backgroundDropdown);
     z-index: @zIndexDropdown;
+    width: 10.125rem !important;
```

Comment:
Is this styling going to affect other components unintentionally?

Reply:
No, it will only affect this dropdown because it is the only component using the `ui-multiselect-menu` class. Confirmed with @andrewkfiedler 👍